package.json

```diff
@@ -57,7 +57,7 @@
     "passport-twitter": "^1.0.2",
     "phantomjs": ">=1.9.0",
     "serve-favicon": "^2.3.0",
-    "socket.io": "^1.3.5",
+    "socket.io": "Automattic/socket.io",
```

Comment:
I agree with @codydaig that we don't want to introduce the development version of socket.io. I think there's a simple solution to this issue. How about we just install the development version of `socket.io-client`? Socket.IO exposes the internal `socket.io-client` that's installed with the package which is exposed to this app via https://github.com/meanjs/mean/blob/master/modules/core/server/views/layout.server.view.html#L52

If we install the stand-alone `socket.io-client` then we can use this version in our tests. One thing I'm not sure of is the impact over how the current `socket.io` exposes `socket.io-client`; my initial guess is that it won't have any impact at all. However, maybe someone else has some insight into that concern.

So the changes to package.json would look like this
```
  "socket.io": "1.3.6",
  "socket.io-client":  "https://github.com/socketio/socket.io-client#master"
```